base/applications/network/ipconfig/lang/ro-RO.rc

```diff
@@ -2,46 +2,47 @@
+    IDS_IPROUTINGYES        "\tRutarea IP activată . . . . . . . : Da\n"
+    IDS_WINSPROXYNO         "\tProxy-ul WINS activat . . . . . . : Nu\n"
+    IDS_WINSPROXYYES        "\tProxy-ul WINS activat . . . . . . : Da\n"
+    IDS_DNSSUFFIXLIST       "\tLista de căutare a sufixului DNS  : %hs\n"
     IDS_EMPTYLINE           "\t                                    %hs\n"
-    IDS_MEDIADISCONNECTED   "\tMedia State . . . . . . . . . . . : Media disconnected\n"
-    IDS_CONNECTIONDNSSUFFIX "\tConnection-specific DNS Suffix. . : %hs\n"
-    IDS_DESCRIPTION         "\tDescription . . . . . . . . . . . : "
-    IDS_PHYSICALADDRESS     "\tPhysical Address. . . . . . . . . : %hs\n"
-    IDS_DHCPNO              "\tDHCP Enabled. . . . . . . . . . . : No\n"
-    IDS_DHCPYES             "\tDHCP Enabled. . . . . . . . . . . : Yes\n"
-    IDS_AUTOCONFIGNO        "\tAutoconfiguration Enabled . . . . : No\n"
-    IDS_AUTOCONFIGYES       "\tAutoconfiguration Enabled . . . . : Yes\n"
-    IDS_IPADDRESS           "\tIP Address. . . . . . . . . . . . : %hs\n"
-    IDS_SUBNETMASK          "\tSubnet Mask . . . . . . . . . . . : %hs\n"
-    IDS_DEFAULTGATEWAY      "\tDefault Gateway . . . . . . . . . : %hs\n"
-    IDS_DHCPSERVER          "\tDHCP Server . . . . . . . . . . . : %hs\n"
-    IDS_DNSSERVERS          "\tDNS Servers . . . . . . . . . . . : %hs\n"
-    IDS_PRIMARYWINSSERVER   "\tPrimary WINS Server . . . . . . . : %hs\n"
-    IDS_SECONDARYWINSSERVER "\tSecondary WINS Server . . . . . . : %hs\n"
-    IDS_LEASEOBTAINED       "\tLease Obtained. . . . . . . . . . : %s\n"
-    IDS_LEASEEXPIRES        "\tLease Expires . . . . . . . . . . : %s\n"
+    IDS_MEDIADISCONNECTED   "\tStadiul media . . . . . . . . . . : Media deconectată\n"
+    IDS_CONNECTIONDNSSUFFIX "\tSufixul DNS specific conexiunii . : %hs\n"
+    IDS_DESCRIPTION         "\tDescriere . . . . . . . . . . . . : "
+    IDS_PHYSICALADDRESS     "\tAdresă fizică . . . . . . . . . . : %hs\n"
+    IDS_DHCPNO              "\tDHCP activat. . . . . . . . . . . : Nu\n"
+    IDS_DHCPYES             "\tDHCP activat. . . . . . . . . . . : Da\n"
+    IDS_AUTOCONFIGNO        "\tAutoconfigurare activată. . . . . : Nu\n"
+    IDS_AUTOCONFIGYES       "\tAutoconfiguraare activată . . . . : Da\n"
+    IDS_IPADDRESS           "\tAdresa IP . . . . . . . . . . . . : %hs\n"
+    IDS_SUBNETMASK          "\tMasca subrețelei. . . . . . . . . : %hs\n"
+    IDS_DEFAULTGATEWAY      "\tPoarta de acces implicită . . . . : %hs\n"
+    IDS_DHCPSERVER          "\tServerul DHCP . . . . . . . . . . : %hs\n"
+    IDS_DNSSERVERS          "\tServerele DNS . . . . . . . . . . : %hs\n"
+    IDS_PRIMARYWINSSERVER   "\tServer WINS primar. . . . . . . . : %hs\n"
+    IDS_SECONDARYWINSSERVER "\tServer WINS secundar. . . . . . . : %hs\n"
+    IDS_LEASEOBTAINED       "\tÎnchirerea obținută . . . . . . . : %s\n"
+    IDS_LEASEEXPIRES        "\tÎnchirerea expiră . . . . . . . . : %s\n"
```

Comment:
"Închirerea" looks like a typo to me. Also, the term "închiriere" (which I presume you intended to use) in the two rows above looks merely as a literal translation. Please do not forget that a good job is translation **_and adaptation_** (for the given context). Therefore, in order to come up with a quality terms, I suggest you to understand a bit more about the notions at hand. One explaining source in this case is this: [what is DHCP lease](https://helpdeskgeek.com/networking/what-is-dhcp-lease-time-and-how-to-change-it/)